### DISCUSSION TITLE
Will chakra-ui break my style?

### DISCUSSION THREAD

**Discussion by quirkybird** [chakra-ui/chakra-ui#8728] [Q&A] [answered]
I used chakra-ui "@chakra-ui/react": "^1.8.9" to inherit into the code I already have. I wrapped my app with `ChakraProvider`, like this:
```jsx
    <ChakraProvider>
          <App />
    </ChakraProvider>
```
Nothing else, but my style has all changed, may I ask why? However, I have not installed the emotion library, and I used styed-components before. May I ask if this is the impact of chakra-ui on the app? What can I do to restore that I still need to use chakra-ui

**Comment by segunadebayo** [comment, marked as answer]
`ChakraProvider` injects a CSS reset. That is why your styled-components look changed.

Scope Chakra to a subtree, or turn the reset off in your system config. Also install `@emotion/react` as a peer. This thread is v1, which we no longer ship for.
